docs/src/index.md

```diff
@@ -76,8 +76,23 @@ The package is in Julia's package list. Install it using this command:
 
 ## Running tests
 
-To run tests, just run the `runtests.jl` file in the `test` folder:
-
-```bash
-$julia runtests.jl
+Press `]` to go into package manager mode and then
+```
+test Agents
 ```
+
+
+## Alterantive Julia package: DynamicGrids.jl
+Agents.jl targets complicated agent based models (ABMs) that are defined on truly arbitrary complex graphs.
+Because of this, the core datastructure of this package is a dictionary that maps unique IDs to Agents, as shown in the [Tutorial](@ref) and specifically in [`AgentBasedModel`](@ref).
+As all agents are unique entities, if one "dies" it is entirely and forever removed from memory.
+Similarly, when a new agent becomes "alive", this literally means that a new agent datastructure is initialized and added to this dictionary.
+
+This design decision allows for arbitrary ABMs and also allows flexible implementation on arbitrary graphs, as well as makes the data collection functionality of Agents.jl powerful and intuitive.
+But this is not necessary for *cellular-automata-like* models, where the grid cell and the "agent" identity are fully equivalent.
+This is the case e.g. in the [Forest fire](@ref) and [Game of life](@ref) examples.
+For such applications the Julia package [DynamicGrids.jl](@ref) can give up to 2 orders of magnitude performance gain than Agents.jl (see `/test/benchmark/DynamicGrids`), while also allowing advanced visualization.

```

Comment:
Rewording: can be up to 2 orders of magnitude faster than Agents.jl

Reply:
I'll modify this a bit in general.